anko/idea-plugin/xml-converter/src/org/jetbrains/kotlin/android/xmlconverter/Util.kt

```diff
@@ -23,9 +23,9 @@ import org.jetbrains.kotlin.android.attrs.readResource
 
 private val INTENT = "    "
 
-internal val attrs = Gson().fromJson(readResource("attrs.json"), Attrs::class.java)
+internal val attrs = Gson().fromJson(readResource("../attrs.json"), Attrs::class.java)
```

Comment:
I'm not too familiar with how java's `File` class determines its abstract pathname, but this was trying to read the resource at `/code/github/anko/anko/idea-plugin/xml-converter/attrs.json`, when it should be `/code/github/anko/anko/idea-plugin/attrs.json` (and likewise for `views.json`). I'm not sure if the abstract pathname is only set to `/code/github/anko/anko/idea-plugin/xml-converter` during the tests, but I'm not sure how this was working in production in the past.